### ISSUE TITLE
Have outputs update while sliders are being slid

### ISSUE THREAD

**Issue by ghost** [CMU-CREATE-Lab/create-lab-visual-programmer#204]
Outputs in the expression builder only update once you 'let go' of a slider. It would be nice to have it update as you're sliding it.

**Comment by chrisbartley** [comment]
Agree, but probably will need to only periodically (e.g. every N milliseconds) send commands while dragging to avoid a flood of commands.  I feel like we tried this early on, but the naive method of sending a command for every change event was too much

Punting and only sending the command once the user lets go was easier at the time due to other things having higher priority.  But good that this is getting some attention now.

**Comment by ghost** [comment]
Yea, the way I did this in iOS when we were testing out BLE was to send commands every time the slider changed by a certain amount and send the final command when the user lets go. I'll play around with it and see what works the best
